share/jupyterhub/static/less/login.less

```diff
@@ -52,6 +52,11 @@
     border-radius: @border-radius-large @border-radius-large 0 0;
   }
 
+  h1 {
+    font-size: 1.03em;
```

Comment:
1.03? Not 1?

Maybe it should be `.auth-form-header > h1` to keep it specific.

Reply:
1.03 was the closest to the previous size. But I can make it 1em, if that's better

Okay

Reply:
Left the comment [earlier](https://github.com/jupyterhub/jupyterhub/pull/4274#discussion_r1053305552), but it should be in `share/jupyterhub/static/less/variables.less`

Reply:
I don't think any changes to the padding are needed. I think you might be getting misled by WAVE increasing the size by inserting its warning markers. The padding is already correct at `10px 20px` and I think it shouldn't change here. This is what it looks like in the main branch:

<img width="504" alt="Screenshot 2022-12-21 at 14 04 18" src="https://user-images.githubusercontent.com/151929/208912172-1f34c301-0987-45cb-9b63-f7b6aca10f08.png">

vs increasing padding to 20px:


<img width="468" alt="Screenshot 2022-12-21 at 14 06 21" src="https://user-images.githubusercontent.com/151929/208912397-c9f07454-87ce-4d2d-bf86-39d036230e48.png">

Reply:
Okay. I will stick to 10px then